### PR TITLE
Readme update

### LINKED ISSUE #3: Feature: Update ReadMe
**Is your feature request related to a problem? Please describe.**
Current repo's ReadMe file is not so descriptive for newcomers and active devs.

**Describe the solution you'd like**

- Make "Binder" header more "modern",
- Add ReadMe's  labels, like in StrayexOS repo, (To Be Consulted)
- Table of Content,
- Tech Stack,
- "Getting Started" section,
- Contribution guidelines (links to Wiki),
- Contact,

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by marekksiazek**
## Summary

Update readme

## Details

Change README content. Add some items like badges and change text.

### CODE REVIEW

**Comment by StraykerPL** [comment]
Some changes are shared with other branch, please watch out for this situation in future as this is dangerous:
image

**Comment by StraykerPL** [comment]
Hehe XD
image

File: README.md [hunk 1]
```diff
@@ -1,19 +1,36 @@
 # Binder
 
-Cross-platform solution for task-oriented data management, supporting Windows, Linux, macOS, Android and iOS. Basic tasks are built up with name, start date and time and category, for organising your tasks. Every tab in UI represents single category. You can sync data via centralised server or between devices via LAN connection. Binder is great to manage your projects pages on GitHub Projects - notes, columns made easy.
+<p align="left">
+    <img src="https://img.shields.io/badge/Version%3A-1.0.0-brightgreen" alt="version_badge" width="120px" height="120px"/>
+    <img src="https://img.shields.io/badge/Coverage-0%25-lightgrey" alt="version_badge" width="120px" height="120px"/>
+    <img src="https://img.shields.io/badge/Platform-windows%20%7C%20macos%20%7C%20linux-lightgrey" alt="version_badge" width="120px" height="120px"/>
+</p>
+Cross-platform solution for **task-oriented data management**, supporting **Windows, Linux, macOS, Android and iOS**.
```
Comment (StraykerPL) [inline README.md]: This formatting is not working:
image

**Review by StraykerPL — CHANGES_REQUESTED** [review]

**Review by marekksiazek — COMMENTED** [review]
Corrected

File: README.md [hunk 2]
```diff
@@ -1,19 +1,37 @@
...
+1.Make sure you have installed .NET Core Runtime at least version 3.1.
 
-For active state of development, there's only one, Windows-based version of Binder. All sync options are planned on future development. GitHub access is not yet available. Description is prepared for final project state - version 1.0.0 of all clients and server solution.
+2.Than just download the newest stable version from [Release](https://github.com/StraykerPL/Binder/releases) page.
+_For detailed info look into user's documentation._
 
-# Contact
+## Documentation
+
+- [Getting Started][quickstart]
```
Comment (StraykerPL) [inline README.md]: Link redirection syntax in Markdown is: [My Website](https://straykersoftware.pl)

Comment (StraykerPL) [reply README.md]: Check once more the order of elements in ticket and rearrange them in ReadMe

Comment (StraykerPL) [reply README.md]: Every dot in ticket is one section in ReadMe, sections' titles should be in table of content.

Comment (StraykerPL) [reply README.md]: Tech Stack is missing.

Comment (StraykerPL) [reply README.md]: Contribution Guidelines are missing.

**Review by StraykerPL — CHANGES_REQUESTED** [review]

**Review by marekksiazek — COMMENTED** [review]
Corrected

**Review by StraykerPL — APPROVED** [review]

**Pull request merged**